src/main/java/Library.java

```diff
@@ -566,4 +568,26 @@ public static int findLevenshteinDistance(String word1, String word2) {
       return stream.collect(Collectors.toList());
     }
   }
+
+  /**
+   * Convert InputStream to String.
+   *
+   * @param inputStream InputStream to convert
+   * @return String
+   * @throws IOException if an I/O error occurs
+   */
+  public static String inputStreamToString(InputStream inputStream) throws IOException {
+    var stringBuilder = new StringBuilder();
+
+    try (var reader = new InputStreamReader(inputStream)) {
```

Comment:
We can wrap `InputStreamReader` inside [BufferedReader](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/BufferedReader.html) to enhance performance